advisor-backend/routes/events.js

```diff
@@ -1,9 +1,56 @@
 import express from "express";
 import { EventDetail } from "../models/event.js";
+import { Op } from "sequelize";
 // number of elements to be displayed on the page
 const pageLimit = 8;
 const router = express.Router();
 
+const parseAndCreateLocationQuery = (query) => {
+  let cities = [];
+  let states = [];
```

Comment:
These could both be `const`, for Object types (which includes arrays in JS) you can still add/remove elements, since you're not reassigning the actual object itself.